rules/do-you-know-how-to-use-tags-for-github-issues/rule.md

```diff
@@ -1,41 +1,34 @@
 ---
 type: rule
-archivedreason: 
 title: Do you know how to use labels for GitHub Issues?
-guid: c530e5c8-1fca-4a4b-bc17-fb0ba65b8625
 uri: do-you-know-how-to-use-tags-for-github-issues
-created: 2019-11-15T04:19:40.0000000Z
 authors:
-- title: Adam Cogan
-  url: https://ssw.com.au/people/adam-cogan
-- title: Andreas Lengkeek
-  url: https://ssw.com.au/people/andreas-lengkeek
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan
+  - title: Andreas Lengkeek
+    url: https://ssw.com.au/people/andreas-lengkeek
 related: []
 redirects:
-- do-you-know-how-to-use-labels-for-github-issues
-
+  - do-you-know-how-to-use-labels-for-github-issues
+created: 2019-11-15T04:19:40.000Z
+archivedreason: null
+guid: c530e5c8-1fca-4a4b-bc17-fb0ba65b8625
 ---
+It is important that you, especially a developer, knows how to use labels for GitHub issues when using an open source project on GitHub, as it would help compact issues and make the issue management workflow more efficient. Essentially, having such a predictable workflow will let the community feel professional.
 
-It is important that you know how to use labels for GitHub issues when using an open source project on GitHub.
-
-
-
-
-Managing your labels on GitHub issues will let the community feel professional and make it easier to work with as it is predictable
+<!--endintro-->
 
+Every new repository comes in with some default labels out of the box that you could use to label your issues to help create a standard workflow in a repository. A list of the default labels and their general uses can be found here: <https://docs.github.com/en/enterprise-server@3.1/issues/using-labels-and-milestones-to-track-work/managing-labels>.
```

Comment:
Hey @jianganna7 

Please fix the link here by using the right Markdown syntax ;)

Reply:
PS. Ping me on Teams in in doubt

Reply:
> Please fix the link here by using the right Markdown syntax ;)

Done - Link fixed using the right Markdown syntax as per https://www.ssw.com.au/rules/rule. Thanks for reviewing :)